packages/charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { classNamesFunction, IClassNames } from '../../Utilities';
+import { IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './DonutChart.types';
+import { Pie } from './Pie/Pie';
+
+const getClassNames = classNamesFunction<IDonutChartStyleProps, IDonutChartStyles>();
+
+export class DonutChartBase extends React.Component<IDonutChartProps, {}> {
+  public static defaultProps: Partial<IDonutChartProps> = {
```

Comment:
Where is this consumed?

Reply:
It's a React construct. If you define this, you don't need to explicitly set props in the constructor, it's done for you automatically. https://reactjs.org/docs/react-component.html#defaultprops